plugin/core/types.py

```diff
@@ -71,6 +72,37 @@ def matches_pattern(path: str, patterns: Any) -> bool:
     return False
 
 
+def sublime_pattern_to_glob(pattern: str, is_directory_pattern: bool, root_path: Optional[str] = None) -> str:
+    """
+    Convert a Sublime Text pattern (http://www.sublimetext.com/docs/file_patterns.html)
+    to a glob pattern that utilizes globstar extension.
+    """
```

Comment:
Should this be used in windows.py as well?

Reply:
How would you use it there? Convert to glob and then how match the glob to file path?